Library/Homebrew/rubocops/cask/homepage_url_trailing_slash.rb

```diff
@@ -27,7 +27,12 @@ def on_homepage_stanza(stanza)
 
           return unless url&.match?(%r{^.+://[^/]+$})
 
-          domain = URI(url_node.str_content).host
+          domain = begin
+            URI(url_node.str_content).host
+          rescue
+            nil
+          end
+          return if domain.blank?
```

Comment:
```suggestion
          domain = begin
            URI(url_node.str_content).host
          rescue URI::InvalidURIError
            nil
          end
          return if domain.blank?
```

If that doesn't work on its own, then maybe also try `include HelperFunctions` and use `string_content(url_node, strip_dynamic: true)` instead of `url_node.str_content`.